### DISCUSSION TITLE
Post-new module hook

### DISCUSSION THREAD

**Discussion by scotje** [puppetlabs/pdk#1027] [Feature Request]
_From @dan-wittenberg on August 22, 2017 16:21_

It would be handy to have a way to expose a hook for doing post-create hooks, such as auto-check into git, etc.

_Copied from original issue: puppetlabs/pdk#266_

**Comment by scotje** [comment]
_From @james-stocks on August 23, 2017 9:4_

Hi Dan -

I created an internal ticket to track this: https://tickets.puppetlabs.com/browse/PDK-444

Thanks
 \- James

**Comment by bmjen** [comment]
Initial idea is to set up a configuration option to set up hooks at the user level config. Could include pre-create and post-create hooks.
